sql/parser/builtins.go

```diff
@@ -294,7 +294,7 @@ var builtins = map[string][]builtin{
 
 	"translate": {stringBuiltin3(func(s, from, to string) (Datum, error) {
 		const deletionRune = utf8.MaxRune + 1
-		translation := make(map[rune]rune, len(from))
+		translation := make(map[rune]rune, utf8.RuneCountInString(from))
```

Comment:
A micro-micro optimization given that the map will expand as needed. I think I'd revert this part of the change.